src/app/leverage/components/your-tokens.tsx

```diff
@@ -57,56 +58,69 @@ export function OpenPositions() {
     }
   }
 
-  if (tokens.length === 0) return null
-
   return (
     <div className='border-ic-gray-600 w-full rounded-3xl border bg-[#1C2C2E] lg:max-w-[67%]'>
-      <h3 className='text-ic-white p-6 font-bold'>Open Positions</h3>
+      <h3 className='text-ic-white p-6 font-bold'>Your Tokens</h3>
       <div className='border-ic-gray-600 flex w-full border-b px-6 pb-3'>
-        <div className='text-ic-gray-400 w-1/2 sm:w-1/3 md:w-3/12'>
-          Position
+        <div className='text-ic-gray-400 w-1/2 sm:w-1/3 md:w-3/12'>Token</div>
+        <div className='text-ic-gray-400 w-1/2 sm:w-1/3 md:w-4/12'>
+          USD Value
         </div>
-        <div className='text-ic-gray-400 w-1/2 sm:w-1/3 md:w-4/12'>Size</div>
         <div className='text-ic-gray-400 hidden md:block md:w-3/12'>
           Current Leverage
         </div>
         <div className='hidden sm:block sm:w-1/3 md:w-2/12'>
-          <span className='sr-only'>Close Position</span>
+          <span className='sr-only'>Close column</span>
```

Comment:
Is this the title of the column? Close column sounds a lil weird, what if we just leave it empty? The buttons should make clear what action can be triggered here.

Reply:
yeah - this is the header for column holding the close buttons...but it's hidden from view and only 'seen' by screen readers (`sr-only` - https://tailwindcss.com/docs/screen-readers#screen-reader-only-elements). Without something there, it's just blank space, so this is a quick and dirty way to make it more accessible/tell the user what's in the column below it if they're using a screen reader

Reply:
oh yea missed the `sr-only` 👍